### ISSUE TITLE
Translate wrongly for JQuery TypeScript

### ISSUE THREAD

**Issue by avatar21** [Microsoft/TypeScript#11850]
**TypeScript Version:**  2.0.3 / nightly (2.1.0-dev.201xxxxx)

**Code**

```ts
// A *self-contained* demonstration of the problem follows...
import jq from "jquery";
jq('#pnlTest').text(apple.eat());
```

**Expected behavior:**
const jquery_1 = require("jquery");
jquery_1('#pnlTest').text(apple.eat());
**Actual behavior:** 
const jquery_1 = require("jquery");
jquery_1.default('#pnlTest').text(apple.eat());

**Comment by kitsonk** [comment]
These issues should be raised at [DefinitelyTyped/DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped) as they are third party typings not directly managed by this repository.

**Comment by aluanhaddad** [comment]
@kitsonk That doesn't effect the TypeScript emit, the behavior here has to do with `allowSyntheticDefaultImports` and misunderstandings related ES6 code consuming commonjs when an interop aware environment does a mapping like `exports.default = module.exports`.

At any rate the emitted code is correct, and if it TypeChecks, then `allowSyntheticDefaultImports` is `true` because jquery.d.ts uses 
```ts
declare var $: JQueryStatic; 
export = $;
```

**Comment by aluanhaddad** [comment]
@avatar21 the translation is correct. Because the syntax 
```ts
import $ from 'jquery';
```
is sugar for the precisely equivalent
```ts
import { $ as default } from 'jquery';
```
which is precisely and correctly transpiled to
```js
const $_1 = require("jquery");
$_1.default('#pnlTest').text(apple.eat());
```
when the `module` setting is `commonjs`, which it is by default.

**Comment by HerringtonDarkholme** [comment]
It is still confusing for new comers to TypeScript, especially they have experience with Babel.
I also encounter these problems when using new library of which the document is written in ES6 (because they also support commonjs). 

If you are using commonjs, I still think `import $ = require('jquery')` is the least confusing one. Though it has been deprecated.

**Comment by RyanCavanaugh** [comment]
Default imports import the default. That's what it means.
